apps/nova/src/components/TutorProfile/LoadingTutorInfo.tsx

```diff
@@ -0,0 +1,18 @@
+import { useFormatMessage } from "@litespace/luna/hooks/intl";
+import { Typography } from "@litespace/luna/Typography";
+
+export const LoadingTutorInfo = () => {
+  const intl = useFormatMessage();
+  return (
+    <div className="h-full flex flex-col justify-center items-center gap-10 mt-20">
+      <div className="w-[80px] h-[80px] animate-spin flex items-center relative rounded-full justify-center bg-[conic-gradient(from_180deg_at_50%_50%,#1D7C4E_0deg,rgba(17,173,207,0)_360deg)]">
```

Comment:
Why do we have to do it this way?

Reply:
When I tried to copy as svg it wasn't being copied correctly, and I tried multiple times
So, I thought it won't be hard to recreate it using HTML

Reply:
This was a new style of loading so I thought it was a one off thing.
When I saw it reused in the dashboard, I made it reusable in luna
But I've done it again with the error here.